### PR TITLE
[16.0][MIG] product_code_mandatory

### PR DESCRIPTION
**Pull request by ced-adhoc**
Last PR closed for the migration module: [#1575](https://github.com/OCA/product-attribute/pull/1575).
The closed PR included changes that were incorporated and improved in this PR. 
Also please find the mig pr to v17 [#1697](https://github.com/OCA/product-attribute/pull/1697)

### CODE REVIEW

**Review by florenciafrigieri2 — APPROVED** [review]
Functional review, working ok. The internal reference is automatically fulfilled for the products and variants if it is empty.

**Comment by jcadhoc** [comment]
@pedrobaeza Hi Pedro, Same case in this pr, i think we can merge.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The migration commit contains changes that don't belong here, and the second commit, if they are migration reviews, should be squashed into the migration commit.

**Comment by ced-adhoc** [comment]
> The migration commit contains changes that don't belong here, and the second commit, if they are migration reviews, should be squashed into the migration commit.

@pedrobaeza 
Reverted changes that don't belong to the module product_code_mandatory in the migration commit.

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ addon | version | maintainers | summary
 [product_code_unique](product_code_unique/) | 16.0.1.0.1 |  | Set Product Internal Reference as Unique
 [product_company_default](product_company_default/) | 16.0.1.0.0 |  | Product Company Default
 [product_cost_security](product_cost_security/) | 16.0.2.2.0 | [![sergio-teruel](https://github.com/sergio-teruel.png?size=30px)](https://github.com/sergio-teruel) [![rafaelbn](https://github.com/rafaelbn.png?size=30px)](https://github.com/rafaelbn) [![yajo](https://github.com/yajo.png?size=30px)](https://github.com/yajo) | Product cost security restriction view
+[product_cost_security](product_cost_security/) | 16.0.1.2.0 | [![sergio-teruel](https://github.com/sergio-teruel.png?size=30px)]
```
Comment (pedrobaeza) [inline README.md]: I still see this incorrect change.

Comment (ced-adhoc) [reply README.md]: Hi @pedrobaeza , it's fixed now, thanks

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**